ui/packages/shared/components/src/Callgraph/index.tsx

```diff
@@ -87,7 +87,7 @@ const Callgraph = ({graph, sampleUnit, width}: Props): JSX.Element => {
               const targetNode = nodes.find(n => n.id === edge.target);
               return (
                 <Edge
-                  key={`edge-${edge.source}-${edge.target}`}
+                  key={`edge-${edge.source as string}-${edge.target as string}`}
```

Comment:
`edge` should be typed earlier, then casting will not be needed. It will make more sense and will have to be done anyway once `noImplicitAny` is enabled (#1579)